tappsbt/decode_test.go

```diff
@@ -3,19 +3,26 @@ package tappsbt
 import (
 	"bytes"
 	"encoding/base64"
+	"encoding/hex"
+	"os"
+	"path/filepath"
 	"reflect"
 	"strings"
 	"testing"
 
 	"github.com/lightninglabs/taproot-assets/address"
 	"github.com/lightninglabs/taproot-assets/asset"
+	"github.com/lightninglabs/taproot-assets/commitment"
 	"github.com/lightninglabs/taproot-assets/internal/test"
+	"github.com/lightninglabs/taproot-assets/mssmt"
 	"github.com/stretchr/testify/require"
 )
 
 var (
 	generatedTestVectorName = "psbt_encoding_generated.json"
 
+	packetHexFileName = filepath.Join(testDataFileName, "packet.hex")
```

Comment:
It might be useful to add a comment here to describe how the contents of `packet.hex` was derived (whether it is a real world example or not).

Reply:
Makes sense, added.